### PR TITLE
Favourite beers screen

### PR DESCRIPTION
**Pull request by alexfts**

### CODE REVIEW

File: client/js/navigation/BottomNavigationLayout.js [hunk 1]
```diff
@@ -68,8 +68,8 @@ const CardTabScreens = createMaterialTopTabNavigator(
 
 const AllEventsTabScreens = createMaterialTopTabNavigator(
   {
-    "All Events": AllEventsTab,
-    "Your Events": FavEventsTab
+    'All Events': AllEventsTab,
+    'Your Events': FavEventsTab
```
Comment (siddharthparmar7) [inline client/js/navigation/BottomNavigationLayout.js:72]: you should not have spaces between your screen ID. Please make the changes.

File: client/js/screens/FavBeers/FavBeers.js [hunk 2]
```diff
@@ -1,17 +1,82 @@
...
+    name: 'removeBeerFromFavourites',
+    options: () => ({
+      refetchQueries: [
+        {
+          query: USER_QUERY
+        }
+      ]
+    })
+  }),
+  withNavigation
```
Comment (siddharthparmar7) [inline client/js/screens/FavBeers/FavBeers.js]: Pass navigation prop instead of using ```withNavigation```

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes. Thank you.

**Review by siddharthparmar7 — APPROVED** [review]

File: client/js/screens/FavBeers/FavBeers.js (hunk 2 above)
Comment (alexfts) [reply client/js/screens/FavBeers/FavBeers.js]: removed withNavigation alltogether because navigation prop is not used

**Pull request merged**